src/main/java/step3/study/controller/RacingGameController.java

```diff
@@ -0,0 +1,53 @@
+package step3.study.controller;
+
+import step3.study.domain.RequestRacingDTO;
+import step3.study.domain.ResponseRacingDTO;
+import step3.study.domain.Car;
+import step3.study.domain.Cars;
+import step3.study.view.InputView;
+import step3.study.view.ResultView;
+
+import java.util.Scanner;
+
+public class RacingGameController {
+    private final InputView inputView;
+    private final ResultView resultView;
+    private final Scanner scanner;
+
+    public RacingGameController() {
+        inputView = new InputView();
+        resultView = new ResultView();
+        scanner = new Scanner(System.in);
+    }
+
+    public void run() {
+        RequestRacingDTO requestRacingDTO = showInputView();
+        ResponseRacingDTO responseRacingDTO = startGame(requestRacingDTO);
+        showResultView(responseRacingDTO);
+    }
+
+    private void showResultView(ResponseRacingDTO responseRacingDTO) {
+        resultView.showResultView(responseRacingDTO.getCars());

```

Comment:
```suggestion
    private void showResultView(ResponseRacingDTO responseRacingDTO) {
        resultView.showResultView(responseRacingDTO);
```

DTO 는 "Data Transfer Object" 입니다. DTO 내부의 값이 아니라, DTO 객체 자체를 View 혹은 도메인에게 넘겨줄 수 있도록 해주세요.